src/test/java/com/aws/greengrass/componentmanager/plugins/docker/DockerImageArtifactParserTest.java

```diff
@@ -141,13 +141,23 @@ void GIVEN_docker_image_artifact_uri_WHEN_valid_THEN_succeed() throws Exception
                 Registry.RegistryType.PUBLIC,
                 Registry.RegistrySource.OTHER);
 
-        assertAll(getImage("docker:image_name"),
-                "image_name",
-                "latest",
+        assertAll(getImage(
```

Comment:
I confused `assertAll` with junit's `Assertions .assertAll`. Maybe rename it like `assertGetImageResult`